mozetl/taar/taar_locale.py

```diff
@@ -65,6 +65,8 @@ def get_addons(spark):
           AND value['app_disabled'] = FALSE -- exclude compatibility disabled addons
           AND value['is_system'] = FALSE -- exclude system addons
           AND locality <> 'null'
+          AND key NOT IN ('loop@mozilla.org', 'firefox@getpocket.com', 'e10srollout@mozilla.org',
```

Comment:
I think the right way to proceed here would be to check the AMO database: system, private addons and unlisted addons (e.g. the ones similar to the ubuntu system integration) will be discarded. We won't have to maintain yet-a-new blacklist and could re-use the logic in the other upcoming jobs. See [here](https://github.com/mozilla/telemetry-batch-view/blob/master/src/main/scala/com/mozilla/telemetry/ml/AddonRecommender.scala#L147) for how we do this in the collaborative filtering job.

Reply:
Yup, 100% agree, this is a much more elegant (and future-proof) solution.

I have added a commit. Please dont review just yet, test coverage comming soon :)